.github/workflows/examples-esp32.yaml

```diff
@@ -61,53 +61,69 @@ jobs:
                    .environment/gn_out/.ninja_log
                    .environment/pigweed-venv/*.log
             - name: Build example All Clusters App
-              timeout-minutes: 25
-              run: scripts/examples/esp_example.sh all-clusters-app
+              timeout-minutes: 10
+              run: scripts/examples/esp_example.sh all-clusters-app sdkconfig_m5stack.defaults
```

Comment:
M5Stack has an ESP32 inside it. So there's no separate need to compile sdkconfig.defaults. M5Stack has the extra display component.

So, if the compilation works fine for M5Stack then it should work fine for ESP32 Devkit as well